lib/mkmf.rb

```diff
@@ -1957,11 +1957,33 @@ def fmt.%(x)
         pkgconfig = nil
       end
       if pkgconfig
+        has_ms_win_syntax = false
+        if $mswin
+          has_ms_win_syntax = xpopen([pkgconfig, "--help"]).read.include?('msvc-syntax')
+          if has_ms_win_syntax
+            args << "--msvc-syntax"
+          else
+            Logging.message("WARNING: #{pkgconfig} does not support the --msvc-syntax. Try using a recent pkgconf instead")
+          end
+        end
```

Comment:
This seems no longer needed.

```suggestion
```

Reply:
Could you explain why/how this is not needed anymore please? Thank you

Reply:
https://github.com/ruby/ruby/pull/9815/files#diff-3e65e2e886eec8afd3714a64d5a673de4e276d0461299c052562892d7176d9f2R1995-R2006
Now arguments of `-l` and `-L` are always built using `LIBARG` and `LIBPATHFLAG`, even if they are not in msvc-syntax.